packages/vaadin-icon/src/vaadin-icon.js

```diff
@@ -83,7 +83,7 @@ class IconElement extends ThemableMixin(ElementMixin(PolymerElement)) {
        */
       size: {
         type: Number,
-        value: 16
+        value: 24
```

Comment:
Would it make sense if the public `icon.size` property was `undefined` by default and the effective value for it was an internal default: `__computeViewBox(size = 24)`?

Reply:
This is not how Polymer observers and computed functions work. 
The problem is that `__computeViewBox` will be only called when `size` is set.

Reply:
Right, the computed function wouldn't invoke if the property was `undefined` by default. So the alternatives would be: 1. use `null` as the default instead or 2. have a separate `__effectiveSize` property that the computed function uses instead.

But the real question here kind of got lost with a not-so-well-thought-out implementation proposal.